src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx

```diff
@@ -224,16 +194,6 @@ describe('DetailsViewCommandBar', () => {
         expect(rendered.getElement()).toMatchSnapshot();
     });
 
-    test('renders with save assessment button', () => {
```

Comment:
These tests were removed because they are not testing the DetailsViewCommandBar and were not moved to their respective test files because they do not cover anything novel.